CONTRIBUTING.md

```diff
@@ -14,26 +14,27 @@ Found a security issue? Please disclose it responsibly. We have a team of develo
 
 ## Contributing to translations
 
-We use [WebTranslateIt][webtranslateit] to manage translation. Please read [our wiki page][translation-wiki] to know more about it. Except if your language isn't on webtranslateit, please do **not** open a pull request to update translations.
+We use [WebTranslateIt][webtranslateit] to manage translations of the app interface. Please read [our wiki page][translation-wiki] to find out more about this. If your language is on WebTranslateIt, please do **not** open a pull request to update translations. If you want to contribute to a language that is not featured on WebTranslateIt, please contact our developers to find out how to proceed.
```

Comment:
I was thinking also about how to handle a language that is not yet part of D* at all. I don't think we have any instructions for that. Let's add a section to the wiki article about what to do for a completely new language, and then the final sentence in the snippet above can be changed to:

> If you want to contribute to a language that is not featured on WebTranslateIt, please read the wiki article above to find out how to proceed.